yt_dlp/extractor/motherless.py

```diff
@@ -148,102 +177,110 @@ def _real_extract(self, url):
         }
 
 
-class MotherlessGroupIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?motherless\.com/gv?/(?P<id>[a-z0-9_]+)'
+class MotherlessPaginatedIE(InfoExtractor):
+    _PAGE_SIZE = 60
+
+    def _correct_path(self, url, item_id):
+        pass
+
+    def _extract_entries(self, webpage, base):
+        for mobj in re.finditer(r'href="[^"]*(?P<href>/[A-F0-9]+)"\s+title="(?P<title>[^"]+)',
+                                webpage):
+            video_url = urllib.parse.urljoin(base, mobj.group('href'))
+            video_id = MotherlessIE.get_temp_id(video_url)
+
+            if video_id:
+                yield self.url_result(video_url, MotherlessIE, video_id, mobj.group('title'))
+
+    def _real_extract(self, url):
+        item_id = self._match_id(url)
+        real_url = self._correct_path(url, item_id)
+        title = self._search_regex(
+            r'^([\w\s]+)\s+', self._html_extract_title(self._download_webpage(real_url, item_id)),
+            'title', default=None)
+
+        def _get_page(idx):
+            page = idx + 1
+            query = {}
+            if idx:
+                query['page'] = page
+
+            current_page = self._download_webpage(real_url, item_id, query=query,
+                                                  note=f'Downloading page {page}')
+            yield from self._extract_entries(current_page, real_url)
+
+        return self.playlist_result(
+            OnDemandPagedList(_get_page, self._PAGE_SIZE), item_id, title)
```

Comment:
avoid downloading the first page twice
inline the playlist title, and regex seemed pretty inexact, better to use `utils.remove_end`
```suggestion
        webpage = self._download_webpage(real_url, item_id, 'Downloading page 1')

        def get_page(idx):
            page = idx + 1
            current_page = webpage if not idx else self._download_webpage(
                real_url, item_id, note=f'Downloading page {page}', query={'page': page})

            yield from self._extract_entries(current_page, real_url)

        return self.playlist_result(
            OnDemandPagedList(get_page, self._PAGE_SIZE), item_id,
            remove_end(self._html_extract_title(webpage), ' | MOTHERLESS.COM ™'))
```

Reply:
Looks great. Added.